doc/guides/security-steward-on-off-boarding.md

```diff
@@ -0,0 +1,29 @@
+# Security Steward Onboarding/OffBoarding
+
+## Onboarding
+
+* Confirm the new steward agrees to keep all private information confidential
+  to the project and not to use/disclose to their employer.
+* Add them to the security-stewards team in the GitHub nodejs-private
+  organization
+* Add them to the standard team in H1 using this
+  [page](https://hackerone.com/nodejs/team_members);
+* Add them to the
+  [jenkin-admins team](https://GitHub.com/orgs/nodejs/teams/jenkins-admins)
+  in the GitHub nodejs org. This is needed for them to be able
+  to lock/unlock the CI during a security release.
```

Comment:
I don't object to this, but this is a change -- currently locking/unlocking the CI for a security release is documented as being something to request the build team to do (see the template issue text for "Notify build-wg of upcoming security release date by opening an issue in [nodejs/build](https://github.com/nodejs/build/issues) to request WG members are available to fix any CI issues." in https://github.com/nodejs/node/blob/master/doc/guides/security-release-process.md).

Reply:
Ok that makes sense to me.  I'll remove that part for now.